engine/grpc/service/hash.go

```diff
@@ -159,3 +104,165 @@ func setValue(s *hash, key string, field interface{}, r *ghash.GHashSetRequest)
 		return false, fmt.Errorf("unknown value type")
 	}
 }
+
+func (s *hash) HExists(ctx context.Context, req *ghash.GHashExistsRequest) (*ghash.GHashExistsResponse, error) {
+	exists, err := s.dbh.HExists(req.Key, req.Field)
+	if err != nil {
+		return nil, err
```

Comment:
Do not use nil to return, use an empty structure to return, otherwise a bug will occur.

Reply:
Later on, too